pom.xml

```diff
@@ -140,7 +140,7 @@
                             <goal>compile</goal>
                         </goals>
                         <configuration>
-                            <release>8</release>
+                            <release>11</release>
```

Comment:
Just FYI, the min version 11 had to be added because of the method `is.readAllBytes();` which is not present in 8. Since the minimum version is now set at 11, we can bump it

Reply:
Just FYI, if we decide that this needs to be backported to Elytron 1.15.x, we'll need to adjust this when backporting.

Reply:
+1